lib/routes.json

```diff
@@ -4990,7 +4990,22 @@
           "type": "string",
           "required": true,
           "description": "The branch (or git ref) you want your changes pulled into. This should be an existing branch on the current repository. You cannot submit a pull request to one repo that requests a merge to a base of another repo."
-        }
+        },
+	"title": {
+	  "type": "string",
+	  "required": true,
+	  "description": "The title of the pull request."
+	},
+	"body": {
+	  "type": "string",
+	  "required": false,
```

Comment:
just leave out `required` unless it's true

Reply:
removed this line